config/deploy.js

```diff
@@ -6,7 +6,6 @@ module.exports = function (deployTarget) {
     build: {},
 
     git: {
-      enabled: true,
```

Comment:
The value defaults to `true` when the `CI` environment variable is set.

https://github.com/dfreeman/ember-cli-deploy-git-ci/tree/v1.0.1#configuration